src/dos/cdrom_image.cpp

```diff
@@ -36,62 +33,65 @@
 #if !defined(WIN32)
 #include <libgen.h>
 #else
-#include <string.h>
+#include <cstring>
 #endif
 
 #include "drives.h"
 #include "support.h"
 #include "setup.h"
 
+// Allow MinGW to use its C11 functions instead of mscvrt
+#if defined(__GNUC__) && defined(__MINGW32__)
+#define __USE_MINGW_ANSI_STDIO 1
```

Comment:
this seems to be worthless.. strip it from configure.ac and here.  fall back to int cast to make mscvrt happy.

Reply:
OK; as I mentioned earlier, we can try `_POSIX_C_SOURCE` - but it doesn't need to happen as part of this review.

Reply:
Trying ..

Reply:
Still not liking the `%zu`; will fall back to the `static<int>` cast for now.